services/app-api/src/emailer/formatters/formatEmailAddresses.test.ts

```diff
@@ -91,17 +91,40 @@ describe('pruneDuplicateEmails', () => {
                 '"Bar 1" <bar@foo.com>',
                 '"Bar 2" <bar@foo.com>',
             ],
-            ['foo@bar.com', 'bar@foo.com'],
+            ['"Foo Bar 1" <foo@bar.com>', '"Bar 1" <bar@foo.com>'],
         ],
-        // multiple aliased email addresses that do not duplicate a raw email string elsewhere in list will remain
-        // this is because there is not way to determine which to prefer
+        // if there are multiple aliased emails, keep the first one
```

Comment:
Nice, I like the clarity this little adjustment brings. And that we can test for it